File: forever.cpp
#include <unistd.h>
#include "iostream"
#include "stdio.h"
using namespace std;

int main() {
    int pid = getpid();
    int num = 1;
    int printNum = pid;

    while (num <= 10000) {
        cout<<"pid_"<<num<<" forever "<<printNum;
        num++;
        printNum = pid * num;
    }
    return 0;
}

File: forever.py
import os
def forever():
    pid = os.getpid()
    num = 1
    printNum = int(pid)
    while num:
        print("pid_" + str(num) + " forever " + str(printNum))
        num += 1
        printNum = int(pid) * num

forever()

